### DISCUSSION TITLE
EditDialog and Editor

### DISCUSSION THREAD

**Discussion by oblist7** [serenity-is/Serenity#6496] [Help] [answered]
![image](https://user-images.githubusercontent.com/60620893/183484390-407384a5-8f47-4514-ad27-38053cd2680b.png)
An unhandled exception occurred while processing the request.
InvalidProgramException: Cook_book.RecipeDB.Forms.RecipeСompositionForm has a [BasedOnRow(typeof(Cook_book.RecipeDB.RecipeСompositionRow), CheckNames = true)] attribute but its 'Fullname' property doesn't have a matching field with same property / field name in the row.

Please check if property is named correctly.

To remove this validation you may set CheckNames to false on [BasedOnRow] attribute.

To disable check for this specific property add a [IgnoreName] attribute to the property itself.
Serenity.PropertyGrid.DefaultPropertyItemProvider.GetPropertyItemsFor(Type type) in DefaultPropertyItemProvider.cs, line 70

**Comment by VictorTomaili** [comment, marked as answer]
Did you read the exception? It's describes everything what you need.
